piptools/exceptions.py

```diff
@@ -13,6 +13,10 @@ def __str__(self):
             'Could not find a version that matches {}'.format(self.ireq),
             'Tried: {}'.format(', '.join(str(version) for version in sorted_versions) or '(no version found at all)')
         ]
+        if sorted_versions:
+            lines.append('You probably have incompatible dependencies.')
```

Comment:
I would try to take a more neutral tone, as this seems to be accusing. How about: 
There are incompatible versions in solved versions of dependencies

Reply:
You're right, I'll change that

Reply:
Updated, I rephrased a bit